l10n_it_declaration_of_intent/models/declaration.py

```diff
@@ -162,7 +162,8 @@ def _check_available_amount(self):
             if declaration.available_amount < 0:
                 raise UserError(
                     _(
-                        "Limit passed for declaration {number}.\nExcess value: {available_amount}{symbol}"
+                        "Limit passed for declaration {number}."
+                        "\nExcess value: {available_amount}{symbol}"
```

Comment:
Nel mondo anglosassone il simbolo della valuta va prima della cifra. Poi sarà cura del traduttore metterlo dopo invertendo i placeholder.

```suggestion
                        "Limit passed for declaration {number}."
                        "\nExcess value: {symbol}{available_amount}"
```